src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -127,7 +127,5 @@ public void sizeIncrease() throws InterruptedException {
             assertNotNull(podSpec);
             assertEquals(podSpec.getPodType(), "p1.x2");
         });
-
-        waitUntilIndexStateIsReady(indexName);
```

Comment:
We do this in an `@AfterEach` call so it's not really necessary to call it here.